utils/prometheus/src/lib.rs

```diff
@@ -1,18 +1,19 @@
-// Copyright 2019 Parity Technologies (UK) Ltd.
 // This file is part of Substrate.
 
-// Substrate is free software: you can redistribute it and/or modify
-// it under the terms of the GNU General Public License as published by
-// the Free Software Foundation, either version 3 of the License, or
-// (at your option) any later version.
-
-// Substrate is distributed in the hope that it will be useful,
-// but WITHOUT ANY WARRANTY; without even the implied warranty of
-// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-// GNU General Public License for more details.
-
-// You should have received a copy of the GNU General Public License
-// along with Substrate.  If not, see <http://www.gnu.org/licenses/>.
+// Copyright (C) 2019-2021 Parity Technologies (UK) Ltd.
```

Comment:
Shouldn't this be GPL? It's not runtime code.

Reply:
Same for the rest of the files in this crate.

Reply:
Idk 😂 I will change it

Reply:
I checked it and it was APACHE before as well. Check `/prometheus/src/networking.rs`

Reply:
Yeah, but it was wrong 🙈

Reply:
Primitives are also apache2